From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Kanaka Raju Nayana <kanaka.raju.nayana@intel.com>
Date: Fri, 23 Feb 2024 10:42:26 +0530
Subject: xe/xe_module: Add debug print

Add debug prints in xe_module.c file to identify
the backport driver.

V1: update with backport tag [Raju]

Signed-off-by: Kanaka Raju Nayana <kanaka.raju.nayana@intel.com>
---
 drivers/gpu/drm/xe/xe_module.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/gpu/drm/xe/xe_module.c b/drivers/gpu/drm/xe/xe_module.c
index 65a7b99..4345089 100644
--- a/drivers/gpu/drm/xe/xe_module.c
+++ b/drivers/gpu/drm/xe/xe_module.c
@@ -206,7 +206,7 @@ static void xe_call_exit_func(const struct init_funcs *func)
 static int __init xe_init(void)
 {
 	int err, i;
-
+	printk(KERN_INFO "XE Backport init, Module is backported from " CPTCFG_BACKPORTS_RELEASE_TAG "\n");
 	for (i = 0; i < ARRAY_SIZE(init_funcs); i++) {
 		err = xe_call_init_func(init_funcs + i);
 		if (err) {
-- 
2.43.0
